src/app/utils/item.js

```diff
@@ -178,7 +179,7 @@ function LibraryItem() {
    */
   this.mapItem = (item = {}, bib) => {
     const id = item && item['@id'] ? item['@id'].substring(4) : '';
-    const itemSource = item.idNyplSourceId ? item.idNyplSourceId['@type'] : undefined;
+    const itemSource = item.idNyplSourceId ? item.idNyplSourceId['@type'] : '';
```

Comment:
Why are we changing this?

Reply:
In this PR, I used `mapItem` for the item fixtures in tests to more accurately reflect what the code is doing. This change and the one you highlighted below came out of that change. 
I don't see anywhere in the codebase for which `itemSource` being `undefined` is relevant. It usually ends up being set to an empty string anyway.